src/Resolve/graphtype.jl

```diff
@@ -570,6 +570,19 @@ function wipe_snapshots!(graph::Graph)
     return graph
 end
 
+function color_string(c, str...)
+    if get(stderr, :color, false)  #check if color is allowed
+        string(Base.text_colors[c], str..., Base.text_colors[:default])
+    else
+        string(str...)
+    end
+end
+
+const CONTRAST_COLORS =  [21:51; 55:119; 124:142; 160:184; 196:220];
+pkgID_color(pkgID) = CONTRAST_COLORS[1 + hash(pkgID) % end]  # Give each package a probably unique color
```

Comment:
I think we should limit to the system colors since that's what we do in Base Julia for error messages. Likely means that we would use a counter to pick the color since hashing would too often give packages with the same color.

Reply:
this is sad, they are such ugly colors by default.

Reply:
So the system colors:
![image](https://user-images.githubusercontent.com/5127634/84597909-ec218500-ae5e-11ea-8c16-de576c35df9d.png)

 - 1 (red): No, that is for warnings/errors
 - 2 (green): No, that is for success
 - 3 (off yellow): yes
 - 4 (blue): No, too low contrast with black
 - 5 (purple): yes
 - 6 (light blue): yes
 - 7 (light grey): No, too low contrast with white
 - 8 (dark grey): No too low contrast with black
 - 9 (orange): Yes
 - 10 (light green): yes
 - 11 (yellow): No, too low contrast with white
 - 12 (indigo): No, too low contrast with black
 - 13 (magenta): yes
 - 14 (cyan): no too low contrast with white
 - 15 (light grey): no too low contrast with white
 - 16 (black): no too low contrast with black.


 So the colors we have are:
 - 3 (off yellow): yes
 - 5 (purple): yes
 - 6 (light blue): yes
 - 9 (orange): Yes
 - 10 (light green): yes
 - 13 (magenta): yes

6 colors is probably enough, I guess

Color contrast being something I am distinctly aware of, being that I have pretty bad vision.
And the other colors are hard for me to read.
Though I don't have cataracts or another condition where contrast *really* starts to matter.

Reply:
I guess if have to do this we can at least prioritize so best colors come first.
Which would be:
```
[5,9,6,3,10,13,    4,1,2,12, 11]
```
(Still avoiding the greys)

Reply:
Ok I had that a bit wrong amended list will be in the next PR